docs/package.json

```diff
@@ -3,17 +3,18 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "docusaurus": "docusaurus",
```

Comment:
don't use NODE_ENV for that, do not modify other scrips, only `dev` script should be added, and old ones shoud stay as they were